test/countries_vat_lists/andorre.vat.js

```diff
@@ -0,0 +1,12 @@
+export const name = 'Andorre'
+
+export const codes = ['AD', 'AND', '020']
+
+export const valid = [
```

Comment:
Honestly if you can provide more valid VATs - it would be really nice

Reply:
ok, I'm gonna work in this. I'll update the pull request after